test/bom.bats

```diff
@@ -258,14 +258,18 @@ EOF
     [ -f .stacker/artifacts/parent/inventory.json ]
     # sbom for this image
     [ -f .stacker/artifacts/parent/parent.json ]
-    if [ -n "${ZOT_HOST}:${ZOT_PORT}" ]; then
+    if [ -n "${ZOT_HOST}${ZOT_PORT}" ]; then
       stacker publish --skip-tls --url docker://${ZOT_HOST}:${ZOT_PORT} --tag latest
       refs=$(regctl artifact tree ${ZOT_HOST}:${ZOT_PORT}/parent:latest --format "{{json .}}" | jq '.referrer | length')
       [ $refs -eq 2 ]
       refs=$(regctl artifact get --subject ${ZOT_HOST}:${ZOT_PORT}/parent:latest --filter-artifact-type "application/spdx+json" | jq '.SPDXID')
       [ $refs == \"SPDXRef-DOCUMENT\" ]
     fi
 
+    if [ -z "${ZOT_HOST}${ZOT_PORT}" ]; then
+    skip "second half of test requires running zot"
```

Comment:
missing a tab? whitespace issue?

Reply:
hrm. looks like emacs shell-script mode's auto indent gets confused in bats files